src/database/config/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why do you need two configs? It's really confusing.

Reply:
My fault. Probably, I forgot to use all the necessary info from this config.